### PR TITLE
ACM-3087 not-showing-blocked-subscription

### PR DESCRIPTION
**Pull request by jeswanke**
Signed-off-by: John Swanke <jswanke@redhat.com>
CleanShot 2023-04-25 at 14 05 42@2x

### CODE REVIEW

File: frontend/public/locales/en/translation.json [hunk 1]
```diff
@@ -1047,6 +1047,7 @@
   "Current page": "Current page",
   "Current version": "Current version",
   "Current window status is": "Current window status is",
+  "Currently Blocked": "Currently Blocked",
```
Comment (KevinFCormier) [inline frontend/public/locales/en/translation.json:1050]: Use lowercase for `b` in "Blocked"

**Comment by KevinFCormier** [comment]
/cc @fxiang1 @chenz4027 

Requested a string change, but Feng and/or Maggie should review the logic.

**Review by fxiang1 — DISMISSED** [review]
Thanks John! This looks really good. The new icon matches the time window feature really well :)

**Review by chenz4027 — DISMISSED** [review]
Love the new icon John!

**Pull request merged**